tests/llm/utils/mock_dal.py

```
@@ -55,30 +55,30 @@ def get_resource_instructions(
-    def get_runbook_catalog(self) -> Optional[List[RobustaRunbookInstruction]]:
+    def get_skill_catalog(self) -> Optional[List[RobustaSkillInstruction]]:
         file_path = self._get_fixture_file_path("runbook_catalog")
         if file_path.exists():
             try:
                 with open(file_path, "r") as f:
                     data = json.load(f)
                     if isinstance(data, list):
-                        return [RobustaRunbookInstruction(**item) for item in data]
+                        return [RobustaSkillInstruction(**item) for item in data]
                     return None
             except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)